Calculator with simple ui.py

```diff
@@ -1,19 +1,59 @@
 # Program make a simple calculator
 
-# This function adds two numbers
 def add(x, y):
+    """
+    This function adds two numbers.
+
+    Examples:
+    >>> add(2, 3)
+    5
+    >>> add(5, 9)
+    14
+    >>> add(-1, 2)
+    1
+    """
     return x + y
 
-# This function subtracts two numbers
 def subtract(x, y):
+    """
+    This function subtracts two numbers.
+
+    Examples:
+    >>> subtract(5, 3)
+    2
+    >>> subtract(9, 5)
+    4
+    >>> subtract(4, 9)
+    -5
+    """
     return x - y
 
-# This function multiplies two numbers
 def multiply(x, y):
+    """
+    This function multiplies two numbers.
+
+    Examples:
+    >>> multiply(4, 2)
+    8
+    >>> multiply(3, 3)
+    9
+    >>> multiply(9, 9)
+    81
+    """
     return x * y
 
-# This function divides two numbers
 def divide(x, y):
```

Comment:
Add error handling if divided by zero.
if num1 = n, where n > 0 and num2 = 0, then error will be returned. Handle it.